src/Billing/X125010837P.php

```diff
@@ -626,7 +626,7 @@ public static function genX12837P($pid, $encounter, &$log, $encounter_claim = fa
         "*" .
         "*" . sprintf('%02d', $claim->facilityPOS()) . ":" . "B" . ":" . $claim->frequencyTypeCode() .
         "*" . "Y" .
-        "*" . "A" .
+        "*" . ($claim->billingFacilityAssignment() ? 'A' : 'C') .
```

Comment:
would make the same change for now on
https://github.com/openemr/openemr/blob/99b01d1525c81948df5d86b8c1c246a80807b98b/src/Billing/X125010837P.php#L2159

even though like you said it maybe moot going forward